packages/ra-ui-materialui/src/detail/TabbedShowLayout.js

```diff
@@ -90,33 +88,41 @@ export class TabbedShowLayout extends Component {
             >
                 <Tabs
                     scrollable
-                    value={this.state.value}
+                    value={location.pathname}
```

Comment:
:+1: smart way of dealing with tab values. I just think you should add a comment leading to line 97 below because I was scratching my head until uI read the rest of the component source.